common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessors.scala

```diff
@@ -170,10 +170,14 @@ trait GettyProcessor {
 }
 
 object GettyXmpParser extends ImageProcessor with GettyProcessor {
-  val excludeFrom = List("newspix international")
+  // including 'newspix internation' because they're sending us lots with that in the credit
+  val excludeFromPatterns = List("newspix international", "newspix internation", "i-images")
+    .map(ex => s"(?i)$ex".r)
```

Comment:
Yea I thought about it but they weren't consistently in one place in the credit, I think it's ok because we're just saying "these aren't Getty" in this block. and sorry for the delay, got home and found our internet was gone >.< thanks for the help